data/brands/shop/fashion_accessories.json

```
@@ -6,6 +6,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "Bottega Veneta",
+      "locationSet": {"include": ["it"]},
```

Based on their website, they have many other locations outside of Italy (https://www.bottegaveneta.com/en-us/storelocator)
```suggestion
      "locationSet": {"include": ["001"]},
```